Package.swift

```diff
@@ -1,15 +1,39 @@
 // swift-tools-version: 5.8
 
 import PackageDescription
+import CompilerPluginSupport
+
+// Availability Macros
+let availabilityTags = [_Availability("AsyncAlgorithms")]
+let versionNumbers = ["1.0"]
+
+// Availability Macro Utilities
+enum _OSAvailability: String {
```

Comment:
naming nit: this doesn't need an underscore since this is not public API (it is restricted to the package)